app/src/main/java/com/example/parkingsystemjava/mvp/listener/CustomListener.java

```diff
@@ -0,0 +1,5 @@
+package com.example.parkingsystemjava.mvp.listener;
+
+public interface CustomListener {
```

Comment:
rename this listener with a representative action or class functionality

Reply:
done